libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -122,12 +122,17 @@ class AP_ICEngine {
 
     enum class Options : uint16_t {
         DISABLE_IGNITION_RC_FAILSAFE=(1U<<0),
+        REDLINE_WARNING = (1U << 0),
     };
     AP_Int16 options;
 
     // start_chan debounce
     uint16_t start_chan_last_value = 1500;
     uint32_t start_chan_last_ms;
+
+    // redline rpm
+    AP_Int16 redline_rpm;
```

Comment:
Would it be worthwhile to just make this AP_Int32 just in case? I feel like 32768 seems a little close to what a turbine rpm could get up to.

Reply:
I did consider that, but I wasn't sure if we had turbine RPM telemetry options. We could easily be over 100k RPM for a turbine.